src/bootstrapper.ts

```diff
@@ -72,12 +81,15 @@ export class Bootstrapper {
         updater: new ReleasePleaseConfig(path, config),
       },
     ];
-    return await this.github.createPullRequest(
-      pullRequest,
-      this.targetBranch,
-      message,
+    return {
+      title: message,
+      body: `Configuring release-please for path: ${path}`,
+      baseBranchName: this.targetBranch,
+      headBranchName: `release-please/bootstrap/${branchName}`,
       updates,
-      {}
-    );
+      number: -1,
```

Comment:
Seems strange?

Reply:
This field is marked as required but is actually only populated if it's an existing PR. We can't really remove it without a breaking interface change. It's on the list to clean up in the next SemVer major release.